mitodl/mitxonline
# Home Page Hero Component

# What are the relevant tickets?
updates https://github.com/mitodl/hq/issues/1677

# Description (What does it do?)
This is a redesign of the Hero component on the mitxonline home page. This redesign will also make the text on the page static/part of the template. This includes both mobile and desktop views as available in the figma. There is a feature flag keeping this from showing outside of dev.



# Screenshots (if appropriate):
![Desktop View](https://github.com/mitodl/mitxonline/assets/7756053/ac2c2b43-ea68-4436-a275-026fe750f58b)
![Mobile View](https://github.com/mitodl/mitxonline/assets/7756053/47847f27-ca6c-4890-943f-05086481b7ac)


# How can this be tested?
Load up the branch in an environment where environment = dev and set up for posthog use with environment variables. You should get the new hero view.  Toggle the flag in posthog to off to make sure the old view still works otherwise.


# Additional Context
flag = [mitxonline-new-featured-hero](https://app.posthog.com/feature_flags/15990)
[Figma](https://www.figma.com/file/Gs4zIhOFv5gVvafawwl6PF/MITx-Online?type=design&node-id=558%3A2716&mode=dev)
